tests/contract_testcases/solana/import_free_function.sol

```diff
@@ -9,3 +9,6 @@ function bar() {
 	x = X.foo();
 	x = X.foo({});
 }
+
+// ---- Expect: diagnostics ----
+// warning: 1:1-29: function can be declared 'pure'
```

Comment:
Does your formatting deal with diagnostics that span over multiple lines?

Reply:
Yes, we use the standard conversion routine so this works fine